launchdarkly-android-client-sdk/src/main/java/com/launchdarkly/sdk/android/ComponentsImpl.java

```diff
@@ -244,30 +244,45 @@ static final class PollingDataSourceBuilderImpl extends PollingDataSourceBuilder
             implements DiagnosticDescription, DataSourceRequiresFeatureFetcher {
         @Override
         public DataSource build(ClientContext clientContext) {
```

Comment:
For reviewers: Updated from using `ClientContext` to `ClientContextImpl` type here to have access to an internal only object, namely the `PerEnvironmentData`.